### ISSUE TITLE
A view is not showing when selecting a vieq

### ISSUE THREAD

**Issue by TanerRiffat** [Innofactor/XrmToolBox.Plugins#46]
I have a view that is not listed when using the open/select view popup in xrmtoolbox.

Not sure why. Only two of the three views are showing. I have made minor changes to the missing view and published several times and it still does not show.

**Comment by Josh-Hoy** [comment]
Hi Taner, I have the same problem. Some Views aren't visible in the View selector after selecting Entity (incident). If I copy the View as a Personal View it appears immediately, but the Public View isn't available. Refreshing Views, multiple Publish did not resolve.

**Comment by TanerRiffat** [comment]
@Josh-Hoy Yeah, pretty sure its broken as I see the same issues. Its open source so anyone could fix it. I was hoping that one of the original authors would fix it though. 

Alternatively you can export the view in a new crm solution (after adding the view first with "add existing component") just for this purpose, then unzip it and edit the customisation.xml file and then rezip again. The view will be in there in xml format, so you could add/delete more filter fields, more sort fields and also delete/add view fields. When done just import the solution. After that then select "add existing component"  and add the view in crm solution you usually work on - if you work from the default solution it will already be there.

I could fix the code but really not inclined as I have a lot on atm.
